arcgis-ios-sdk-samples/Maps/Create and save a map/CreateOptionsViewController.swift

```diff
@@ -143,13 +143,11 @@ class CreateOptionsViewController: UIViewController, UITableViewDataSource, UITa
         let basemap = self.basemaps[self.selectedBasemapIndex].copy() as! AGSBasemap
         
         //create an array of the selected operational layers
-        var layers = [AGSLayer]()
-        for index in self.selectedLayersIndex {
-            let layer = self.layers[index].copy() as! AGSLayer
-            layers.append(layer)
+        let selectedLayers: [AGSLayer]? = selectedLayersIndex.isEmpty ? nil : selectedLayersIndex.map { (index) -> AGSLayer in
```

Comment:
This, because of the ternary operator, is a little hard to follow. Maybe something like this would be more readable?

```swift
let selectedLayers: [AGSLayer]?
if !selectedLayersIndex.isEmpty {
    selectedLayers = selectedLayersIndex.map { layers[$0].copy() as! AGSLayer }
} else {
    selectedLayers = nil
}
```

Reply:
Looks good, thanks